librz/il/definitions/bitvector.c

```diff
@@ -0,0 +1,823 @@
+#include "bitvector.h"
+
+// Core idea : use ut8 to represent 8-bit long vector
+//           -> use ut8 array to implement bit vector
+
+// Assert 1 : only bitvector with the same length can be calculated together
+
+// init
```

Comment:
We should add Doxygen for all `RZ_API` functions here. Moreover, it would be nice to refer them to the original functions from BAP wherever possible.

Reply:
Yes, I'm going to add the description and signatures from BAP to core theory mods, though they are not exported to rizin.